### PR TITLE
[dev] Review documentation

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Removed the readthedocs documentation, transferring it to the README.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
https://github.com/dojot/dojot/issues/1584

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,22 +89,442 @@ docker/waitForDb.py
...
+  "created": "2020-09-16T14:50:09.292714+00:00"
+}
+```
 
-## API Documentation
+All attributes that are read-write can be used when creating or updating the template. All of them
+are returned when retrieving device data. You might also notice some new attributes:
+- `data_attrs`: stores attributes with the types `dynamic`, `static` and `actuator`.
+- `config_attrs`: stores attributes with the type `meta`. You can only create this type of attribute
+  via AP, check its [documentation](https://dojot.github.io/device-manager/apiary_latest.html) for
```
Comment (mprevide) [inline README.md:300]: via API

**Review by thiagodpf — DISMISSED** [review]
:+1:

**Review by thiagodpf — DISMISSED** [review]
:+1:

File: README.md [hunk 2]
```diff
@@ -102,22 +89,442 @@ docker/waitForDb.py
...
+
+__NOTE THAT__ all messages reside in Kafka's `dojot.device-manager.device` topic.
+
+The events that are emitted by the Device Manager are:
+
+- `actuate`
+- `create`
+- `remove`
+- `update`
+- `template.update` **deprecated**
```
Comment (mprevide) [inline README.md:322]: At the end I don't know if we can say that really deprecated.... I think that not...

File: README.md [hunk 3]
```diff
@@ -102,22 +89,442 @@ docker/waitForDb.py
...
+
+### **Event: `actuate`**
```
Comment (mprevide) [inline README.md:324]: In my tests the event `actuate` doest exist, just the event `configure`

Comment (mprevide) [reply README.md:324]: `{"event": "configure", "data": {"attrs": {"t": "100"}, "id": "19c216"}, "meta": {"service": "admin", "timestamp": 1600453556258}}`

Comment (mprevide) [reply README.md:324]: To send a msg to device I use:

```sh
curl -X PUT \
    http://localhost:8000/device/19c216/actuate \
    -H "Authorization: Bearer ${JWT}" \
    -H 'Content-Type:application/json' \
    -d '{"attrs": {"t" : "100"}}'
```

File: README.md [hunk 4]
```diff
@@ -102,22 +89,442 @@ docker/waitForDb.py
...
+- `create`
+- `remove`
+- `update`
+- `template.update` **deprecated**
+
+### **Event: `configure`**
+
+This message is published whenever a device must be configured. Its payload is:
+
+```json
```
Comment (mprevide) [inline README.md:328]: I think you can remove that example block and leave only the next one.

**Review by mprevide — CHANGES_REQUESTED** [review]
I left one more comment
And don't forget about release/v0.5.0

**Review by thiagodpf — APPROVED** [review]

**Review by mprevide — APPROVED** [review]
:smile_cat:

**Review by mprevide — APPROVED** [review]

**Review by canattofilipe — APPROVED** [review]

**Pull request merged**